eng/Packages.props

```diff
@@ -13,6 +13,7 @@
     <PackageVersion Include="Microsoft.CodeAnalysis.Collections" Version="$(MicrosoftCodeAnalysisCollectionsVersion)" />
     <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsVersion)" />
     <PackageVersion Include="Microsoft.IO.Redist" Version="$(MicrosoftIORedistVersion)" />
+    <PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="$(MicrosoftNetCompilersToolsetVersion)" />
```

Comment:
It looks like NuGet 1009 says it's already an implicit reference, but if you want to nowarn that, I'm fine with that.

I'm wondering if you should set some sort of "Included" flag for these packages and only conditionally evaluate this ItemGroup based on whether that flag has been set. I noticed it was included 2-3 times for several downstream projects.

Reply:
This one already had such a flag so I'll adopt it.